src/Lime.Transport.SignalR/SignalRTransport.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Lime.Protocol;
+using Lime.Protocol.Network;
+using Lime.Protocol.Serialization;
+
+namespace Lime.Transport.SignalR
+{
+    /// <summary>
+    /// Base class for SignalR transport implementation.
+    /// </summary>
+    /// <inheritdoc />
+    public abstract class SignalRTransport : TransportBase
```

Comment:
This class does not have anything specific to SignalR. It does not even reference the SignalR namespace. Maybe should have another name? Make it internal?

Reply:
Yep I meant it to be internal, will change

Reply:
oh shoot actually that breaks the build. `ClientSignalRTransport` must be a `public` class, and thus it would have inconsistent accessibility if `SignalRTransport` were `internal`. That's why I didn't do it :/ 

While I do agree with you on principle, I don't think it isn't really a big issue? I mean, it's an `abstract` class. What difference does it really make to the consumer to not have SignalR-specific logic inside it? 🤔